### PR TITLE
Move bugfix.py to core server

### PR DESCRIPTION
**Pull request by swalladge**
Implement `bugfix.py` script fixes in the core server (take 2 due to epic git fail in #68 ).

### CODE REVIEW

**Comment by swalladge** [comment]
Strangely this seems to disable line building altogether... :\

File: feature_server/server.py [hunk 1]
```diff
@@ -205,7 +207,16 @@ def on_block_build_attempt(self, x, y, z):
         return self._can_build()
 
     def on_line_build_attempt(self, points):
-        return self._can_build()
+        if not self._can_build():
```
Comment (yvt) [inline feature_server/server.py]: Surprisingly, `self._can_build()` [returns `None`](https://github.com/piqueserver/piqueserver/blob/master/feature_server/server.py#L198) if building is allowed. This line should be replaced by `if self._can_build() == False:` to get the desired behavior.

Comment (swalladge) [inline feature_server/server.py]: oooh i see. So `on_line_build_attempt` must onnly ever be checked for false as well... weird

Comment (swalladge) [inline feature_server/server.py]: Yep, that was it. It works fine now. :)

File: piqueserver/server.py [hunk 2]
```diff
@@ -211,7 +215,16 @@ def on_block_build_attempt(self, x, y, z):
         return self._can_build()
 
     def on_line_build_attempt(self, points):
-        return self._can_build()
+        if self._can_build() == False:
```
Comment (noway) [inline piqueserver/server.py]: looks solid

**Pull request merged**